pkg/pocket/core/check.go

```diff
@@ -39,11 +40,24 @@ func (c *Core) startCheckConsistency() {
 		waitingForCheck = true
 		t++
 		go func(t int) {
+			var (
+				result bool
+				err    error
+			)
 			log.Info("ready to compare data")
-			result, err := c.checkConsistency(false)
+
+			err = wait.PollImmediate(1*time.Minute, 10*time.Minute, func() (bool, error) {
+				result, err = c.checkConsistency(false)
```

Comment:
This error can be caused by database unavailable either data inconsistency, and retry is necessary is for the first condition only.

Reply:
Seems there will be a panic when data inconsistency occurred?

Reply:
Yes, so there should be only inconsistency and time exceeded errors.

Reply:
Why not add a cause checking here?